litho-core/src/main/java/com/facebook/litho/InternalNode.java

```diff
@@ -250,6 +250,12 @@ InternalNode fullImpressionHandler(
   @Nullable
   String getTestKey();
 
+  /**
+   * Arbitrary data which may be attached to a component for testing.
+   */
+  @Nullable
+  String getTestTag();
```

Comment:
Should't the return value be `Object`?

Reply:
Ah yep, I also noticed a few of the other function signatures were incorrect as well. Thanks for catching this! Fixed.